vars/Ubuntu-16.yml

```diff
@@ -0,0 +1,4 @@
+---
+__certbot_version: "0.26.1-1+ubuntu10.04.1+certbot+2"
```

Comment:
`16.04` instead of `10.04`, right?

Reply:
In any case the new version is `0.28.0-1+ubuntu16.04.1+certbot+4`

Reply:
yes! :sweat_smile: